.circleci/config.yml

```diff
@@ -916,39 +939,100 @@ workflows:
                 context:
                     - mattermost
                 delivery: none
+            - build_web:
+                build_config_name: integreat-test-cms
+                context:
+                    - mattermost
+                name: build_ig_test_cms_web
+                requires:
+                    - bump_version
             - check:
                 context:
                     - mattermost
+                filters:
+                    branches:
+                        ignore:
+                            - main
             - e2e_web:
                 context:
                     - mattermost
+                filters:
+                    branches:
+                        ignore:
+                            - main
                 requires:
                     - bump_version
                     - check
-            - build_web:
-                build_config_name: integreat-test-cms
+            - build_android:
+                build_config_name: integreat-e2e
                 context:
                     - mattermost
-                name: build_ig_test_cms_web
+                    - tuerantuer-google-play
+                    - tuerantuer-android
+                filters:
+                    branches:
+                        ignore:
+                            - main
+                name: build_e2e_android
                 requires:
                     - bump_version
                     - check
+            - e2e_android:
+                context:
+                    - mattermost
+                    - browserstack
+                filters:
+                    branches:
+                        ignore:
+                            - main
+                requires:
+                    - build_e2e_android
+            - build_ios:
+                build_config_name: integreat-e2e
+                context:
+                    - mattermost
+                    - tuerantuer-apple
+                    - tuerantuer-ios
+                filters:
+                    branches:
+                        ignore:
+                            - main
+                name: build_e2e_ios
+                requires:
+                    - bump_version
+                    - check
+            - e2e_ios:
+                context:
+                    - mattermost
```

Comment:
Mattermost context is not needed, as we don't notify in mattermost, when e2e fails on a feature branch, do we?